src/stash/src/postgres.rs

```diff
@@ -507,15 +496,12 @@ impl Postgres {
     async fn consolidate_batch_tx(
         stmts: &PreparedStatements,
         tx: &Transaction<'_>,
-        collections: Vec<(Id, Option<Antichain<Timestamp>>)>,
+        collections: &[Id],
     ) -> Result<(), StashError> {
         let mut futures = Vec::with_capacity(collections.len());
-        for (collection_id, since) in collections {
+        for collection_id in collections {
             futures.push(async move {
-                let since = match since {
-                    Some(since) => since,
-                    None => Self::since_tx(stmts, tx, collection_id).await?,
-                };
+                let since = Self::since_tx(stmts, tx, *collection_id).await?;
```

Comment:
Why are we removing the optional `since`s from a bunch of methods?

Reply:
There were 2 callers of the method, and only one of them was using the since, and that one usage got removed here, so now there were zero callers that already knew the since.